prism_compile.c

```diff
@@ -951,6 +951,11 @@ pm_compile_index_write_nodes_add_send(bool popped, LINK_ANCHOR *const ret, rb_is
         }
         ADD_SEND_WITH_FLAG(ret, &dummy_line_node, idASET, argc, INT2FIX(flag));
     }
+    else if (keywords && keywords->keyword_len) {
+        ADD_INSN1(ret, &dummy_line_node, opt_reverse, INT2FIX(keywords->keyword_len + block_offset + 1));
```

Comment:
Will this fall back if optimizations are turned off?

Reply:
`opt_reverse` is used to reverse the stack elements and it's used even when optimizations are turned off.